### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by adamwoodnz**
Several things were broken and needed to change in order for this workflow to pass again:

1. WP Env was not starting because Docker Compose was not available; fixed by upgrading WordPress scripts and wp-env. This required a Node upgrade to 20 (consistent with our other repos).
2. XDebug was being loaded by default and was not compatible with PHP 7.4; fixed by upgrading WordPress scripts and wp-env. This required a Node upgrade to 20 (consistent with our other repos).
3. PHPUnit Polyfills were a missing required dependency; fixed by adding to composer and then fixing issues with test setUp() hooks.
4. 2 tests were failing as actual output of the shortcode didn't match the expected output; fixed by replacing `strftime` with `DateTime`.

Tests pass!

### Testing
Best tested in sandbox. There should be no change to functionality.
1. Regression test the shortcode, which is used on the Make homepage to render meeting times for each team.
Screenshot 2024-10-14 at 10 20 19 AM
2. Regression text the Make Meetings calendar

### CODE REVIEW

File: includes/wporg-meeting-posttype.php [hunk 1]
```diff
@@ -864,7 +866,7 @@ public function meeting_time_shortcode( $attr, $content = '' ) {
 				$out .= '<time class="date" date-time="' . esc_attr( $next_meeting_iso ) . '" title="' . esc_attr( $next_meeting_iso ) . '">' . $next_meeting_display . '</time> ';
 				$out .= sprintf( esc_html__( '(%s from now)', 'wporg-meeting-calendar' ), human_time_diff( $next_meeting_timestamp, current_time( 'timestamp' ) ) );
 				if ( $post->location && $slack_channel ) {
-					$out .= ' ' . sprintf( wp_kses( __( 'at <a href="%1$s">%2$s</a> on Slack', 'wporg-meeting-calendar' ), array( 'a' => array( 'href' => array() ) ) ), 'https://wordpress.slack.com/messages/' . $slack_channel, $post->location );
+					$out .= ' ' . sprintf( wp_kses( __( 'accessible via <a href="%1$s">%2$s</a> on Slack or  <a href="%3$s">%4$s:community.wordpress.org</a> on Matrix', 'wporg-meeting-calendar' ), array( 'a' => array( 'href' => array() ) ) ), 'https://wordpress.slack.com/messages/' . $slack_channel, $post->location, '/' . strtolower( 'Core Performance' === $attr['team'] ? 'performance' : $attr['team'] ) . '/chat/', $post->location );
```
Comment (adamwoodnz) [inline includes/wporg-meeting-posttype.php:869]: This change comprises to commits made directly in SVN and not synced back to GitHub. See https://dotorg.trac.wordpress.org/browser/wordpress/website/wp-content/plugins/wporg-meeting-calendar/includes/wporg-meeting-posttype.php?rev=20906 and https://dotorg.trac.wordpress.org/browser/wordpress/website/wp-content/plugins/wporg-meeting-calendar/includes/wporg-meeting-posttype.php?rev=21316

**Comment by dd32** [comment]
> WP Env was not starting because Docker Compose was not available; fixed by adding Docker and Docker compose install steps

Does updating the wp-env version resolve that? `docker-compose` was replaced by `docker compose` which I think a later version of wp-env works with.

**Comment by adamwoodnz** [comment]
> Does updating the wp-env version resolve that?

Yes! Thanks for that, removed those steps.

**Review by dd32 — APPROVED** [review]
Visual review of diff LGTM

**Pull request merged**